.ci/pytorch/build.sh

```diff
@@ -276,6 +276,10 @@ else
   ( ! get_exit_code python setup.py clean] )
   ( ! get_exit_code python setup.py clean bad_argument )
 
+  if [[ -n "$TRITON_VERSION" ]]; then
+    checkout_install_triton_at_commit "$TRITON_VERSION"
```

Comment:
What is blocking us from running this inside Docker image build as the commit pin https://github.com/pytorch/pytorch/blob/main/.ci/docker/ci_commit_pins/triton.txt is already there?  IMO, only stuffs like vision needs to be built and installed at CI build time because they depend on PyTorch

Reply:
Sorry this was a bit premature of a review, I'll put the context in the summary and request a another review.